manifests/plugin/discovery/params.pp

```diff
@@ -1,8 +1,8 @@
 # Default parameters for foreman::plugin::discovery
 class foreman::plugin::discovery::params {
-  $version         = 'latest'
-  $source          = 'http://downloads.theforeman.org/discovery/releases/latest/'
-  $initrd          = "foreman-discovery-image-${version}.el6.iso-img"
-  $kernel          = "foreman-discovery-image-${version}.el6.iso-vmlinuz"
+  $major_version   = 'latest'
+  $minor_version   = 'latest'
+  $source          = "http://downloads.theforeman.org/discovery/releases/${major_version}/"
+  $image_name      = "fdi-image-${minor_version}.tar"
```

Comment:
Well, I think we either just expose versions and the (base) source URL, then concatenate the full URL, or we supply the full URL as a parameter.  I suppose how often are people going to want to override the version, and would it better to expose just that, or to override the whole URL?